mithril-client-cli/src/commands/cardano_db/list.rs

```diff
@@ -32,6 +32,14 @@ impl CardanoDbListCommand {
     /// Main command execution
     pub async fn execute(&self, context: CommandContext) -> MithrilResult<()> {
         let client = client_builder_with_fallback_genesis_key(context.config_parameters())?
+            .with_capabilities(RequiredAggregatorCapabilities::And(vec![
+                RequiredAggregatorCapabilities::SignedEntityType(
```

Comment:
The backend type is available, only the required capability should be asked, else this would fails if either one is missing.

Reply:
Fixed